packages/dds/tree/src/feature-libraries/typed-schema/typedTreeSchema.ts

```diff
@@ -100,22 +101,9 @@ export class FlexMapNodeSchema extends TreeNodeSchemaBase {
 /**
  */
 export class LeafNodeSchema extends TreeNodeSchemaBase<ValueSchema> {
-	public get leafValue(): ValueSchema {
-		return this.info;
-	}
-
 	protected _typeCheck2?: MakeNominal;
-	public static create<const Name extends string, const Specification extends ValueSchema>(
-		builder: Named<string>,
-		name: TreeNodeSchemaIdentifier<Name>,
-		specification: Specification,
-	): LeafNodeSchema {
-		return new LeafNodeSchema(
-			builder,
-			name,
-			specification,
-			new LeafNodeStoredSchema(specification),
-		);
+	public constructor(builder: Named<string>, name: string, specification: ValueSchema) {
```

Comment:
is builder still a reasonable variable name?

Reply:
I don't think its worth refactoring that nomenclature out of the flex-tree logic, since its just going to get deleted, and its still semantically accurate as it records a name of what built it for use in errors.